use-timescale/hypertables/improve-query-performance.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Improve query performance
+excerpt: How and why to use chunk skipping to optimize hypertable performance
+products: [cloud, mst, self_hosted]
+keywords: [hypertables, indexes, chunks]
+---
+
+# Improve query performance
+
+When you execute a query on a hypertable, you do not parse the whole table; you only access the chunks necessary
+to satisfy the query. This works well when the `WHERE` clause of a query uses the column by which a hypertable is 
+partitioned. For example, in a hypertable where every day of the year is a separate chunk, a query for September 1 
+accesses only the chunk for that day. However, many queries use columns other than the partitioning one. For example, 
+a table might have two columns: one for when data was gathered and one for when it was added to the database. If you 
+partition by the date of gathering, a query by the date of adding accesses all chunks in the hypertable and slows the 
+performance.
+
```

Comment:
Good point on diagrams, thank you. For examples, I was aiming at making this a very lean document (as tech docs should be). I also didn't want this to copy your blog post word for word, for SEO and other reasons. It seemed to me that having more examples would not make the point much clearer (again, because this is a tech document, rules are very different for tech marketing content). I can add more, of course, if you feel that having only one does not convey the point. I think it would be better though to just explicitly link to your blog post for an in-depth look.

Reply:
I made the example more specific, linked to your blog post, and added a diagram